src/lambda_function/s3_to_glue/app.py

```diff
@@ -0,0 +1,87 @@
+import os
+import json
+import boto3
+import logging
+import datetime
+from dateutil import parser
+
+logger = logging.getLogger()
+logger.setLevel(logging.INFO)
+
+BUCKET_NAME = 'recover-pilot-data'
+WORKFLOW_NAME = 'S3_to_JSON'
+
+
+def query_files_to_submit(objects : dict,
+                          files_to_submit : list,
+                          trigger_event_date : datetime):
+    ''' Queries the files in bucket for ones with the same
+        date as the trigger event
+    '''
+    files_current = objects["Contents"]
+    logger.info('Querying S3 bucket for new data from source')
+
+    # query bucket for new/updated files
+    for fi in files_current:
```

Comment:
I'm assuming that `files_current` is list of dictionaries?

Reply:
Yup!